src/data/markdown/docs/02 javascript api/02 k6/group- name- fn -.md

```diff
@@ -17,6 +17,19 @@ Run code inside a group. Groups are used to organize results in a test.
 | ---- | ------------------------- |
 | any  | The return value of _fn_. |
 
+<Blockquote mod="warning">
+
+Avoid using `group` with async functions or asynchronous code.
+If you do, k6 might apply tags in a way that is unreliable or unintuitive.
```

Comment:
```suggestion
<Blockquote mod="warning" title="Avoid using group with async operations">

This might cause k6 to apply tags in a way that is unreliable or unintuitive.
```